abe/helper_functions/email_helpers.py

```diff
@@ -195,7 +201,7 @@ def reply_email(to, event_dict):
 
 
 def send_email(server, email_text, sent_from, sent_to):
-    server.sendmail(sent_from, sent_to, email_text)
+    server.sendmail(sent_from, sent_to, email_text.encode('utf-8'))
```

Comment:
It looks like the test case needs to be updated to expect this: https://travis-ci.org/olinlibrary/ABE/builds/372690411